test/unit/lodash.spec.js

```diff
@@ -0,0 +1,690 @@
+                var element = params[1] instanceof Function;
+
+                expect(element).to.eql(true);
+                expect(params.length).to.eql(2);
+            });
+            it('should pass mock test', function () {
+                var testObj = [
+                    {qq: '123', ee: 10, active: false},
+                    {qq: '456', ee: 40, active: true}
+                ];
+                var checkFunc = _.partition.apply(_, [testObj, spy]);
+
+                expect(spy).to.have.been.callCount(2);
+                expect(checkFunc).to.eql([
+                    [{qq: '456', ee: 40, active: true}],
+                    [{qq: '123', ee: 10, active: false}]
+                ]);
+            });
+            it('should split persons to two tables using active property', function () {
+                expect(_.partition.apply(_, datasets.partition(users))).to.eql([
+                    [{name: 'Harry', age: 40, active: true}, {name: 'Francheska', age: 61, active: true}],
+                    [{name: 'Andrinio', age: 36, active: false}, {name: 'Kodi', age: 1, active: false}]
+                ]);
+            });
+        });
+        describe('partition2 function', function () {
+            var params = datasets.partition2(users);
+
+            it('should match types of passing elements', function () {
+                var element = typeof params[1];
+                expect(element).to.eql('string');
+                expect(element).to.have.length.above(1);
+                expect(params.length).to.eql(2);
+            });
+            it('should split persons only to truthy table when predicate is true for all', function () {
+                expect(_.partition.apply(_, datasets.partition2(users))).to.eql([[
+                    {name: 'Andrinio', age: 36, active: false},
+                    {name: 'Harry', age: 40, active: true},
+                    {name: 'Kodi', age: 1, active: false},
+                    {name: 'Francheska', age: 61, active: true}],
+                    []]);
+            });
+        });
+        describe('partition3 function', function () {
+            var params = datasets.partition3(users);
+            it('should match types of passing elements', function () {
+                expect(typeof params[1]).to.be.oneOf(['number', 'string', 'object', 'boolean']);
+            });
+            it('should split persons only to falsy table when predicate is false for all', function () {
+                expect(_.partition.apply(_, datasets.partition3(users))).to.eql([[],
+                    [
+                        {name: 'Andrinio', age: 36, active: false},
+                        {name: 'Harry', age: 40, active: true},
+                        {name: 'Kodi', age: 1, active: false},
+                        {name: 'Francheska', age: 61, active: true}
+                    ]
+                ]);
+            });
+        });
+    });
+
```

Comment:
I don't think that third test scenario is good. You can basically put almost anything there and even if that may test all of the types I stand that we still are missing good test case for objects and arrays. Moreover, why aren't you using object collections at all?